bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Image.java

```diff
@@ -164,9 +164,9 @@ public final class Image extends Resource implements Drawable {
  * as shown in the following example:
  * <pre>
  *    Image i = new Image(device, width, height);
- *    GC gc = new GC(i);
- *    gc.drawRectangle(0, 0, 50, 50);
- *    gc.dispose();
+ *    try (var gc = GC.create(i)) {
+ *      gc.drawRectangle(0, 0, 50, 50));
+ *    }
```

Comment:
Please apply this doc update also to the Image implementations for Linux and Mac.